lib/pages/chat/chat_view.dart

```diff
@@ -269,58 +269,70 @@ class ChatView extends StatelessWidget {
                             if (controller.room!.canSendDefaultMessages &&
                                 controller.room!.membership == Membership.join)
                               Container(
-                                margin: EdgeInsets.only(
-                                  bottom: bottomSheetPadding,
-                                  left: bottomSheetPadding,
-                                  right: bottomSheetPadding,
-                                ),
                                 constraints: const BoxConstraints(
                                   maxWidth: FluffyThemes.columnWidth * 2.5,
                                 ),
                                 alignment: Alignment.center,
                                 child: controller.room?.isAbandonedDMRoom ==
                                         true
-                                    ? Row(
-                                        mainAxisAlignment:
-                                            MainAxisAlignment.spaceEvenly,
-                                        children: [
-                                          TextButton.icon(
-                                            style: TextButton.styleFrom(
-                                              padding: const EdgeInsets.all(16),
-                                              foregroundColor: Theme.of(context)
-                                                  .colorScheme
-                                                  .error,
-                                            ),
-                                            icon: const Icon(
-                                              Icons.archive_outlined,
-                                            ),
-                                            onPressed: controller.leaveChat,
-                                            label: Text(
-                                              L10n.of(context)!.leave,
-                                            ),
-                                          ),
-                                          TextButton.icon(
-                                            style: TextButton.styleFrom(
-                                              padding: const EdgeInsets.all(16),
+                                    ? Padding(
+                                        padding: EdgeInsets.only(
+                                          bottom: bottomSheetPadding,
+                                          left: bottomSheetPadding,
+                                          right: bottomSheetPadding,
+                                        ),
```

Comment:
```suggestion
                                    ? Padding(padding: bottomSheetPadding)
```

Reply:
bottomSheetPadding is a double value, I can't use it there and I don't want a top padding aswell